sdcm/cluster.py

```diff
@@ -2723,7 +2723,7 @@ def tags(self) -> Dict[str, str]:
     def init_log_directory(self):
         assert '_SCT_TEST_LOGDIR' in os.environ
         self.logdir = os.path.join(os.environ['_SCT_TEST_LOGDIR'], self.name)
-        makedirs(self.logdir)
+        os.makedirs(self.logdir)
```

Comment:
make it:
```python
os.makedirs(self.logdir, exist_ok=True)
```

to not fail if directory exists

Reply:
Done